### ISSUE TITLE
Adding Multi-omics data support to ABioTrans Plus

### ISSUE THREAD

**Issue by MoHelmy** [nrnb/GoogleSummerOfCode#161]
### OVERVIEW
[ABioTrans](https://www.frontiersin.org/articles/10.3389/fgene.2019.00499/full) is a bio-statistical/informatics tool that was developed in R to analyze RNASeq gene expression data. [ABioTrans Plus](http://combio-sifbi.org/ABioTrans/) is the webserver of ABioTrans that analyzes RNASeq or Microarray gene expression data and performs multiple gene set and protein set analyses including gene ontology (GO), pathways enrichment and co-expression analysis. ABioTrans Plus is developed to be a one-stop server for transcriptomic data analysis, including the downstream analysis of the set of resultant differentially expressed genes (DEGs).

### Focus
This project aims to develop ABioTrans Plus v3.0 which will support a wider range of high throughput omics data, namely MS-based proteomics and metabolomics data. Furthermore, developing a detailed report with the performed analysis with figures and data ready to be included in the publications. 

### Goals
1- Developing an input module for each of the proteomics and metabolomics data.
2- Developing the preprocessing modules for each data type.
3- Connecting the preprocessed data with the downstream biostatistical analysis e.g PCA and correlation analysis.
4- Create the analysis report generator that generates the report at the end of the analysis. 

### Coding Skills
R programming,  Shiny, JavaScript, HTML/CSS

### Acceptance Criteria
Students interested in working on this project will be asked to provide their CV and school transcript then the short-listed students will be asked to do an evaluation task that will be used for assessing their fitness for the project.

### Public Repository
[ABioTrans at GitHub](https://github.com/buithuytien/ABioTrans)

### Mentors
[Mohamed Helmy](mailto:mohamed_helmy@sifbi.a-star.edu.sg), [Thuy Tien Bui](mailto:Bui_Thuy_Tien@sifbi.a-star.edu.sg)

### Our Lab website
[The COMPUTATIONAL BIOLOGY AND OMICS GROUP](http://combio-sifbi.org)
Singapore Institute of Food and Biotechnology Innovation (SIFBI), A*STAR]

**Comment by yash7520** [comment]
hello this is yash 
i am new to open source and passionate about contributing to this project
but i am not able to understand acceptance criteria do i have to submit resume before contributing to this project

**Comment by MoHelmy** [comment]
@yash7520  Yes, you need to submit a CV/resume and your school transcript.

**Comment by yash7520** [comment]
@MoHelmy      thank you sir 
kindly please tell where i have to submit resume and school transcript

**Comment by MoHelmy** [comment]
@yash7520 to the email addresses of the mentors mentioned above.

**Comment by RATED-R-SUNDRAM** [comment]
Hello @MoHelmy, I am Shivam Sundaram undergraduate in computational biology, I have skills in R programming and various machine language algorithms with front end languages like HTML, CSS, and Js, I am excitedly looking forward to working on this project any guidance on how to get started will be very helpful.

**Comment by RATED-R-SUNDRAM** [comment]
@MoHelmy I've mailed you my resume and transcript hoping to get a word from you soon.

**Comment by theakhiljha** [comment]
Hey, @MoHelmy, I hope you are doing well.
I am Akhil Kumar Jha pre-final year student from Galgotias University (India).
I found this project is very interesting about my stack. I have worked on R shiny app for my project. and also I have worked on data analysis and visualization.
currently, I am looking forward to contributing to open source.
So suggest to me, what I can do further in this project!
I love to hear from you soon!
